Tests/ConcordiumSwiftSDKTests/ClientTests.swift

```diff
@@ -7,6 +7,8 @@ import NIOPosix
 /// Temporary test for exercising the gRPC client. To run, adjust the channel target to point to a running node.
 final class ClientTests: XCTestCase {
     let channelTarget = ConnectionTarget.host("localhost", port: 20000)
+    let someBlockHash = "a21c1c18b70c64680a4eceea655ab68d164e8f1c82b8b8566388391d8da81e41"
+    let someAccountAddr = "35CJPZohio6Ztii2zy1AYzJKvuxbGG44wrBn7hLHiYLoF2nxnh"
```

Comment:
Nitpicking (and the intention is actually pretty clear): The Swift API Design Guidelines suggest to avoid abbreviations (https://www.swift.org/documentation/api-design-guidelines/)

Reply:
No prob.